src/test/java/org/foundation/coding/StatementTest.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Refactor Code Tasks.
+ */
+package org.foundation.coding;
+
+import static org.junit.Assert.assertNotSame;
+import java.util.logging.Logger;
+import org.foundation.coding.customer.Customer;
+import org.foundation.coding.movie.Movie;
+import org.foundation.coding.movie.NewReleaseMovie;
+import org.foundation.coding.movie.RegularMovie;
+import org.foundation.coding.rental.Rental;
+import org.junit.Test;
+
+/**
+ * Test printer message.
+ */
+public class StatementTest {
+  //Log instance.
+  static final Logger LOGGER = (Logger) Logger.getLogger(String.valueOf(StatementTest.class));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make the "LOGGER" logger private static final. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1312)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unnecessary cast to "Logger". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1905)